test/modules/core/webgl1/polyfill-context.spec.js

```diff
@@ -18,3 +19,24 @@ test('WebGL#polyfillContext', t => {
 
   t.end();
 });
+
+
+test('WebGL#polyfillContext getSupportedExtensions when context is lost', t => {
+  const {gl, gl2} = fixture;
+
+  const getSupportedExtensionsSpy = makeSpy(gl, 'getSupportedExtensions');
+  getSupportedExtensionsSpy.returns(null);
+
+  const extensions = polyfillContext(gl);
+  t.ok(extensions, 'extensions were returned');
+
+  if (gl2) {
+    const getSupportedExtensions2Spy = makeSpy(gl2, 'getSupportedExtensions');
+    getSupportedExtensions2Spy.returns(null);
+    const extensions2 = polyfillContext(gl2);
+    t.ok(extensions2, 'extensions were returned');
```

Comment:
Should we remove the spy after we are done? 

The `gl2` context is shared between files to prevent the test machines from running out of contexts.